doc/tech/usecases.md

```diff
@@ -11,3 +11,5 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+5- Use Cases 
```

Comment:
If these are the Edge use cases it should be in a sunchapter, like 5.1 also if we would like to have it in the toc  you should add an anchor to it.

Reply:
@CsatariGergely I have added TOC and an anchor to it.